src/DotLiquid.Tests/ContextTests.cs

```diff
@@ -376,12 +394,19 @@ public void TestAddFilter()
             Context context = new Context(CultureInfo.InvariantCulture);
             context.AddFilters(new[] { typeof(TestFilters) });
             Assert.That(context.Invoke("hi", new List<object> { "hi?" }), Is.EqualTo("hi? hi!"));
-            context.SyntaxCompatibilityLevel = SyntaxCompatibility.DotLiquid22;
-            Assert.That(context.Invoke("hi", new List<object> { "hi?" }), Is.EqualTo("hi? hi!"));
-
+            
             context = new Context(CultureInfo.InvariantCulture);
             Assert.That(context.Invoke("hi", new List<object> { "hi?" }), Is.EqualTo("hi?"));
-            context.SyntaxCompatibilityLevel = SyntaxCompatibility.DotLiquid22;
+        }
+
+        [Test]
+        public void TestAddFilter_NotFoundException()
+        {
+            Context context = new Context(CultureInfo.InvariantCulture) { SyntaxCompatibilityLevel = SyntaxCompatibility.DotLiquid22 };

```

Comment:
```suggestion
            var context = new Context(CultureInfo.InvariantCulture) { SyntaxCompatibilityLevel = SyntaxCompatibility.DotLiquid22 };
```
(I do see that this code has not made use of `var` for joint declaration and assignment - this is only a nit, feel free to ignore)

Reply:
Maybe I'm old school but I have a preference to specify the type if you know it, rather than `var`. That way it's obvious to the reader. I wouldn't think the compiler cares either way (if it can infer the type).

I'm curious when you'd choose `var` over a known type.

Reply:
I've looked into this further - it seems common practice to use `var` for constructor assignment (exactly like `var context = new Context()`) where the type is written on the same line. That seems a good rule of thumb.